solidity/dashboard/src/css/liquidity-page.less

```diff
@@ -29,7 +29,7 @@
     margin: 0.3rem 0;
   }
 
-  .liquidity__info {
+  .liquidity__info, .liquidity__info--locked {
```

Comment:
What do you think about adding some transition effect to the `liquidity__info-tile` eg. `transition: background-color 0.5 ease`? On mainnet fetching data may take a little longer so the user will see the grey color definitely longer and a smooth transition from grey to mint could improve UX.

Reply:
That's actually a great idea! I wouldn't have thought it up better myself! 🔥